lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -600,14 +600,6 @@ def write(chunk)
               @log.info 'Successfully sent gRPC to Stackdriver Logging API.'
             end
 
-          rescue GRPC::Cancelled => error
-            increment_failed_requests_count(GRPC::Core::StatusCodes::CANCELLED)
-            increment_retried_entries_count(entries_count, error.code)
```

Comment:
Might want to propagate this line below (i.e., replace `entries.length` with `entries_count`, since we've bothered computing it).

Reply:
Done.